zdm/parameters.py

```diff
@@ -252,34 +259,49 @@ class WidthParams(data_class.myDataClass):
             "Notation": "w_{\\rm min}",
         },
     )
-
     WNbins: int = field(
-        default=5,
+        default=12,
         metadata={"help": "Number of bins for FRB width distribution", "unit": ""},
     )
+    WNInternalBins: int = field(
+        default=1000,
+        metadata={
+            "help": "Number of internal bins to use for calculation purposes in numerical estimates of the width distribution",
+            "unit": "",
+            "Notation": "",
+        },
+    )
     WMin: int = field(
-        default=0.1,
-        metadata={"help": "Minimum scattering value to model", "unit": "ms"},
+        default=0.01,
+        metadata={"help": "Minimum width value to model", "unit": "ms"},
     )
     WMax: int = field(
         default=100,
-        metadata={"help": "Maximum scattering value to model", "unit": "ms"},
+        metadata={"help": "Maximum width value to model", "unit": "ms"},
     )
 
 
 # FRB intrinsic scattering parameters
 @dataclass
 class ScatParams(data_class.myDataClass):
+    ScatFunction: int = field(
+        default=2,
+        metadata={
+            "help": "Which scattering function to use. 0: log-constant. 1: lognormal. 2: half log-normal",
+            "unit": "",
+            "Notation": "",
+        },
+    )
     Slogmean: float = field(
-        default=0.7,
+        default=-1.3,
```

Comment:
same point as above.  Where do the defaults come from?

Reply:
From the paper :-)